tests/testUserAccount.py

```diff
@@ -248,6 +247,73 @@ def test_user_can_reset_password(self):
         response_data = json.loads(response.get_data().decode('utf-8'))
         self.assertEqual(response_data['message'], 'password updated')
 
+    def test_users_new_password_is_not_less_than_6_characters_long(self):
+        """test that users new passwords are not less than  6 characters long"""
+        user_data = {
+            "email": "meshmbuvi@gmail.com",
+            "username": "musyoka",
+            "password": "mbuv",
+            "confirm password": "mbuv"
+        }
+        response = self.app.put('/api/v1/auth/reset_password',
+                                data=json.dumps(user_data),
+                                content_type='application/json')
+        self.assertEqual(response.status_code, 400)
+        response_data = json.loads(response.get_data().decode('utf-8'))
+        self.assertEqual(response_data['message'], 'passwords should be 6 characters or more.')
+
+    def test_users_email_is_valid(self):
+        """test that users email when changing password is valid"""
+        user_data = {
+            "email": "meshmbuvi",
+            "username": "musyoka",
+            "password": "mbuvi1",
+            "confirm password": "mbuvi1"
+        }
+        response = self.app.put('/api/v1/auth/reset_password',
+                                data=json.dumps(user_data),
+                                content_type='application/json')
+        # self.assertEqual(response.status_code, 400)
+        response_data = json.loads(response.get_data().decode('utf-8'))
+        self.assertEqual(response_data['message'], 'Email is invalid')
+
+    def test_user_passwords_match(self):
+        """test that users email when changing password is valid"""
```

Comment:
The name of this test and the doc string content are contradictory -  are you testing for valid email or for matching passwords?

Reply:
Noted and corrected.